src/System.Private.CoreLib/shared/System/Reflection/Assembly.cs

```diff
@@ -254,7 +254,7 @@ public static Assembly LoadFile(string path)
                 if (s_loadfile.TryGetValue(normalizedPath, out result))
                     return result;
 
-                AssemblyLoadContext alc = new IndividualAssemblyLoadContext();
+                AssemblyLoadContext alc = new IndividualAssemblyLoadContext(String.Format("Assembly.LoadFile({0})", path));
```

Comment:
Nit: The coding convention is to use lower case for built-in types, e.g. this should be `string.Format`.